app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/ComposeActivity.kt

```diff
@@ -151,4 +171,13 @@ class ComposeActivity : ComponentActivity() {
                 navController.navigate(Routes.Profile)
             }
         )
+
+    private fun getTopRoute(route: String) =
+        when (route) {
+            Routes.Home,
+            Routes.Commits,
+            Routes.RepositoryBrowser()
```

Comment:
@br-jared-stockton Agreed.  I'm working on a more generalized version of nav drawer in PathFinder as we type. :).  Once I get that prototyped.  I'll move it to LaunchPad and then retrofit back into Arch Demo.

Reply:
@br-jared-stockton  - Just as an FYI, I believe that the "current route" passed in to navigation composable will always be the generalized version with brackets and arguments will already be parsed out.  At least this is behavior that @LuisRocketRangel observed.  A moot point since this code is going away, but worth noting for the future.